internal/users/localgroups/getpwent_c.go

```diff
@@ -3,24 +3,39 @@ package localgroups
 // #include <stdlib.h>
 // #include <pwd.h>
 // #include <grp.h>
+// #include <errno.h>
+// #include <string.h>
+//
+// void unset_errno(void) {
+//   errno = 0;
+// }
+//
+// int get_errno(void) {
+//   return errno;
+// }
```

Comment:
I feel it's easier to maintain code here when using C-style comments (`/* .... */`) so can go multi-line.

Or move it to a local header when things are getting longer (but not needed here).

Reply:
Accessing to errno should likely be mutex-protected too, or just use a single mutex for both user-names and groups operations

Reply:
> I feel it's easier to maintain code here when using C-style comments

done

Reply:
> Accessing to errno should likely be mutex-protected too, or just use a single mutex for both user-names and groups operations

done